tb_plugin/examples/resnet50_autograd_api.py

```diff
@@ -10,9 +9,6 @@
 import torchvision.models as models
 
 from torch.autograd.profiler import profile
```

Comment:
It will be better to  change to the new profiler namespace and use torch.profiler in all examples going forward

Reply:
We have another example for new profiler namespace that uses torch.profiler in this dir. This example is just for old api. Is it necessary to delete it?